### DISCUSSION TITLE
STM32 NUCLEO-G491RE board

### DISCUSSION THREAD

**Discussion by lamik** [stm32duino/Arduino_Core_STM32#1605] [Variants] [answered]
Hi!

I cannot see the STM32 NUCLEO-G491RE on the list. How to add support for it? I need to use just this board.

**Comment by fpistm** [comment, marked as answer]
Hi @lamik 
You can refer to:
https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-%28board%29
